journalpump/rsyslog.py

```diff
@@ -82,9 +134,9 @@ def close(self):
     def send(self, message):
         if self.socket is None:
             self._connect()
+        self.socket.sendall(message[:self.max_msg-1])
         if len(message) >= self.max_msg:
-            message[self.max_msg - 1] = b'\n'
-        self.socket.sendall(message[:self.max_msg])
+            self.socket.sendall(b'\n')
```

Comment:
This could be optimized a bit but that can be done if it ever turns out to be an issue.